server/services/utils/licenceValidation.js

```diff
@@ -57,6 +57,7 @@ function getMessage(errorType, errorMessage, errorPath) {
         .filter(pathItem => !Number.isInteger(pathItem))
         .join('_');
 
+    // Shouldn't this be first? Custom messages for postcode or telephone don't get used
```

Comment:
Could be. It was originally done this way to give you option for more specific message like 'Enter a date that is not in the past' rather than a catch all message for the field. If we're going to have a defined message for every field then I guess all the stuff above can completely go

Reply:
will come back to it